src/hooks/useDimensions.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+// Use this hooks instead of useMeasure because the results were wrong
```

Comment:
On le référence quelque part useMeasure ? Si faut pas l'utiliser, on peut augmenter des types pour l'indiquer en deprecated.

Reply:
non, useMeasure fait parti de react-hooks

updated
```
// Use this hooks instead of react-hookz/web/useMeasure because the results were wrong
```

Reply:
Ok, j'ai essayé rapidement d'augmenter le type pour le rendre deprecated mais pas réussi, tant pis !